Makefile

```diff
@@ -96,6 +96,12 @@ libtwin.a_cflags-y += $(shell sdl2-config --cflags)
 TARGET_LIBS += $(shell sdl2-config --libs)
 endif
 
+ifeq ($(CONFIG_BACKEND_FBDEV), y)
```

Comment:
You should edit `configs/Kconfig` to select `CONFIG_CURSOR` if mouse support is enabled in Linux framebuffer backend.

See https://pypi.org/project/kconfiglib/